live-hls.js

```diff
@@ -99,6 +109,10 @@ getHeaderObjects = function(fileContent) {
       continue;
     }
 
```

Comment:
I double checked with Joe on style related to this.  I recommended we stick with this implementation since we have no written guide on how to deal with this.